content/en/news/releases/1.5.x/announcing-1.5.3/index.md

```diff
@@ -9,6 +9,11 @@ aliases:
     - /news/announcing-1.5.3
 ---
 
+DO NOT USE this release. USE release 1.5.4 instead.
```

Comment:
```suggestion
**DO NOT USE this release. USE release 1.5.4 instead.**
```

Reply:
Would this be a better use case for a warning?

Reply:
{{< warning >}}
DO NOT USE this release. USE release 1.5.4 instead.
{{< /warning >}}